qa/tasks/cephfs/test_mirroring.py

```diff
@@ -1305,3 +1319,23 @@ def test_local_and_remote_dir_root_mode(self):
         self.disable_mirroring(self.primary_fs_name, self.primary_fs_id)
         self.mount_a.run_shell(["rmdir", "l1/.snap/snap0"])
         self.mount_a.run_shell(["rmdir", "l1"])
+
+    def test_mirror_multiple_peers(self):
```

Comment:
This should also check if the snapshots have been synchronized to all the peers.

I suggest, creating a separate class for
- Multiple peer setup (site-a->site-b, site-a->site-c, site-a->site-d)
- Cascaded setup (site-a->site-b->site-c->site-d)

Mostly, you'd need to verify if the snapshots get synchronized to all peers - this would involve checks via `peer status` command.